src/sentry/snuba/metrics/extraction.py

```diff
@@ -307,8 +307,8 @@ def _get_function_support(function: str) -> SupportedBy:
 
 def _get_args_support(function: str, args: Sequence[str]) -> SupportedBy:
     # apdex can have two variations, either apdex() or apdex(value).
-    if function == "apdex" and len(args) < 2:
-        return SupportedBy.both()
+    if function == "apdex":
```

Comment:
This call will behave wrong in case someone passes apdex with more than 1 param. Are we ok with that?

Reply:
I think we could improve the validation and actually check the number of params based on the function, since I suppose we can have at most one for each function, unless it's a special edge case.

My main concern for this validation is that we saw that people can make HTTP requests with the data how they prefer. I am sure there is upstream validation but I wouldn't trust it.

Reply:
It's not necessary for this PR but I think it's something to consider.